getting-started-service-registry/README.adoc

```diff
@@ -185,9 +185,13 @@ To connect your applications or services to a {registry} instance in the web con
  * *CNCF Schema Registry API* provides compatibility with the CNCF specification
 
 . In the *Connection* page, copy the *Core Registry API* URL, or one of the other API URLs if you are using a different client, to a secure location. This is the server endpoint that you'll need to connect to this {registry} instance.
++
+NOTE: The remainder of this section describes how to create a service account and copy the generated credentials.
```

Comment:
Given the way quick starts are presented as a series of discrete tasks in the Hybrid Cloud Console, referring to _sections_ doesn't feel _quite_ right for that delivery format.

We could either:

- Split the difference between the cloud console and Customer Portal, and generically refer to them as _procedures_.
- Use conditionalization (`ifdef::qs[]`, ifndef::qs[]) to refer to _tasks_ in the cloud console, and _sections_ on the Customer Portal.